app/assets/src/components/ui/labels/PathogenLabel.jsx

```diff
@@ -0,0 +1,24 @@
+import Label from "./Label";
+import React from "react";
+
+const PathogenLabel = ({ type, number }) => {
+  let type2text = {
+    category_A: "priority A pathogen",
```

Comment:
is the "priority A pathogen" labeling more of a prototype thing?

Reply:
yeah. We haven't gotten definitive text yet. We felt the term "pathogenic | A" in the mocks was too vague. I need to restyle the colors too

Reply:
We still set on "Pathogenic | A" right?

Reply:
sure, although it doesn't make any sense

Reply:
It is just a representation we set on. 
I would agree if you say we should explained it somewhere... maybe in a tooltip in the label or in the header of the column.

Reply:
ok changed